sdks/python/src/opik/message_processing/streamer.py

```diff
@@ -39,6 +47,19 @@ def put(self, message: messages.BaseMessage) -> None:
             else:
                 self._message_queue.put(message)
 
+        # Display message in console
+        if isinstance(message, messages.CreateTraceMessage):
+            projects_url = url_helpers.get_projects_url()
+            project_name = message.project_name
+            if (
+                self._project_name_most_recent_trace is None
+                or self._project_name_most_recent_trace != project_name
+            ):
+                LOGGER.info(
+                    f'Started logging traces to the "{project_name}" project at {projects_url}.'
+                )
+                self._project_name_most_recent_trace = project_name
```

Comment:
I'm not entirely sure, but it might be better to place these comparisons and argument assignments under a lock also. However, this could introduce some performance drawbacks.